@navikt/core/react/src/form/combobox/Input/ToggleListButton.tsx

```diff
@@ -11,10 +12,14 @@ export const ToggleListButton = forwardRef<
   ToggleListButtonProps
 >(({ toggleListButtonLabel }, ref) => {
   const { isListOpen, toggleIsListOpen } = useFilteredOptionsContext();
+  const { focusInput } = useInputContext();
   return (
     <button
       type="button"
-      onPointerUp={() => toggleIsListOpen()}
+      onPointerUp={() => {
+        toggleIsListOpen();
+        focusInput();
```

Comment:
I added `focusInput()` here because it seemed like a bug to not focus the input when you toggle the "show list of selections"? 🤔 At least this seems like intended behavior now (post change).

Reply:
Might argue that its as intended for not focusing input. For desktopusers it might be nice to save a click, but for mobile/touch users a virtual keyboard will pop up when a textfield is focused. This might be worse then since if they wanted to type, they would have clicked the input.

Reply:
True, can argue for both. I did some testing just now however, and it seems like the behaviour is already like this on mobile? 🤔 (maybe I'm just not precise enough with my taps, and keep missing the dropdown button). If so it's not going to change anything.

I'd argue that since the arrow is visually "inside" the border rectangle (for the pseudo-input field, I say pseudo since it's a `ul` with list items where one of them is the input field). This is an ok change? I'll remove it if you have a stronger opinion on it ofc.

Reply:
I think I agree about this change. The only issue I see is that the focus is lost while the pointer is down, so that the focus border appears to blink.

I think the argument about the arrow being inside the border makes sense. Even though it might be annoying that the virtual keyboard pops up, you might not understand that you can write/search otherwise.